documentation/restarting.markdown

```diff
@@ -66,3 +66,6 @@ There are three ways to trigger your Puppet Server environment to refresh and pi
 * Changes to [`ca.cfg`](./configuration.markdown#service-bootstrapping) to enable or disable Puppet Server's certificate authority (CA) service.
 
 For these types of changes, you must restart the process by using the operating system's service framework, for example, by using the `systemctl` or `service` commands.
+
+> Note: To ensure that the Puppet master and CA service is running in a platform agnostic way, use the `puppet resource service puppetserver ensure=running` command. This command is equivalent to  `systemctl start puppetserver`. For more information on the resource command and managing a server’s desired state, see 
```

Comment:
I'm sorry, can we say

> This command is equivalent to  `systemctl start puppetserver` _on systems that support it_.

instead of

> This command is equivalent to  `systemctl start puppetserver`.